public/usage-examples/geometry/cicle_intersects_triangle-1-example-opp.cs

```diff
@@ -0,0 +1,46 @@
+using SplashKitSDK;
+
+public class Program
+{
+    public static void Main()
+    {
+        // A fixed triangle is placed as the target.
+        Window gameWindow = new Window("Circle-Triangle Intersect", 600, 600);
```

Comment:
For programs that only require a single window, you can create one without creating a variable for it:

```suggestion
        SplashKit.OpenWindow("Circle Triangle Intersect", 600, 600);
```